iOS_Labs/Utils/Photos/PhotoController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PhotoController.swift
+//  iOS_Labs
+//
+//  Created by Daniil Oleshchuk on 29.11.2021.
+//
+
+import Foundation
+import Alamofire
+
+
+struct PhotoController {
```

Comment:
Why this is `controller`? It is more like a `manager` or `service` or even `helper` as we are using here `static` variables and functions.